Automattic/wp-calypso
# Restore Newspack version in ETK

### Changes proposed in this Pull Request

In #57104, the newspack version in ETK was accidentally removed. This PR restores it and prevents future similar problems from happening. I also noticed that the Newspack blocks version was not set to the same one we install, but to the latest that yarn provides. I updated the logic to restore the way we get the newspack version prior to #55759.

### Testing instructions
This should have no impact on the installed build. Run `yarn build` for ETK locally and verify the version does not change. Run `yarn` after changing the newspack version in package json to 1.33.1. Then run `yarn build` again. The version should change to 1.33.1
